app/src/org/commcare/views/EntityViewTile.java

```diff
@@ -61,71 +62,67 @@
     private final float XLARGE_FONT = getResources().getDimension(R.dimen.font_size_xlarge);
     private final float DENSITY = getResources().getDisplayMetrics().density;
 
-    private final int CELL_PADDING_HORIZONTAL = (int) getResources().getDimension(R.dimen.cell_padding_horizontal);
-    private final int CELL_PADDING_VERTICAL = (int) getResources().getDimension(R.dimen.cell_padding_vertical);
-    private final int ROW_PADDING_HORIZONTAL = (int) getResources().getDimension(R.dimen.row_padding_horizontal);
-    private final int ROW_PADDING_VERTICAL = (int) getResources().getDimension(R.dimen.row_padding_vertical);
-
-    // constants used to calibrate how many tiles should be shown on a screen
+    private final int DEFAULT_TILE_PADDING_HORIZONTAL =
+            (int)getResources().getDimension(R.dimen.row_padding_horizontal);
+    private final int DEFAULT_TILE_PADDING_VERTICAL =
+            (int)getResources().getDimension(R.dimen.row_padding_vertical);
+
+    // Constants used to calibrate how many tiles should be shown on a screen at once -- This is
+    // saying that we expect a device with a density of 160 dpi and a height of 4 inches to look
+    // good in portrait mode with 4 tiles of 6 rows each being displayed at once.
```

Comment:
The preexisting defaults that were being used to calibrate the # of tiles shown on a screen were a bit wonky. 7 was very high for `DEFAULT_NUM_TILES_PER_SCREEN_PORTRAIT` (unless the default was meant to be on a tab? not sure). The preexisting strategy was also not taking absolute screen height into account at all, which was causing case tiles to appear very inconsistently on a phone vs. tablet, regardless of screen density. 

The comment here and the comment above `computeNumTilesPerScreen()` should make it clear what the new strategy I'm using is, but let me know if it's not.